Monika After Story/game/script-stories.rpy

```diff
@@ -39,6 +42,12 @@ init -1 python in mas_stories:
             story.unlocked = True
 
 
+    def unlock_pooled_story(label):

```

Comment:
can you use var name other than `label`? Since its a renpy keyword and we don't want to accidentally overwrite it.